spec/controllers/web/dashboard/users_controller_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Web::Dashboard::UsersController, type: :controller do
+  let(:user) { create(:user, :staff) }
+
+  describe 'GET #index' do
+    context 'not signed in' do
+      before { get :index }
+
+      it 'redirect to root landing' do
+        expect(response).to redirect_to '/'
+      end
+    end
+
+    context 'not authorized' do
+      let(:user) { create(:user, :active) }
```

Comment:
you missed the `login_user` in this context.
It is weird that this spec passes. Please investigate why that happens.

Reply:
I did following the example:
describe Web::Dashboard::TestTaskAssignmentsController do
  let(:user) { create(:user, :staff) }

  describe 'GET #index' do
    context 'not signed in' do
      before { get :index }

      it 'redirects to root landing' do
        expect(response).to redirect_to '/'
      end
    end

    context 'not authorized' do
      let(:user) { create(:user, :active) }
      before { get :index }

      it 'redirects to dashboard root' do
        expect(response).to redirect_to dashboard_root_path
      end
    end